Sources/FirebaseFirestore/FirestoreDataConverter.swift

```diff
@@ -42,4 +44,73 @@ struct FirestoreDataConverter {
       return nil
     }
   }
+
+  /// Maps a Swift version of a Firestore document into a `MapFieldValue` as expected by the
+  /// Firestore set and update functions.
+  ///
+  /// - Parameter document: The dictionary that describes the document of data you'd like to create.
+  /// - Returns: a `MapFieldValue` with converted types that Firestore will understand.
+  static func firestoreValue(document: [String: Any]) -> firebase.firestore.MapFieldValue {
```

Comment:
Could you please make this explicit what the access control is?

```suggestion
  internal static func firestoreValue(document: [String: Any]) -> firebase.firestore.MapFieldValue {
```

Reply:
The entire type is internal, does that not mean that the static functions inside of it also inherit those access controls?

Reply:
Yes, though, making it explicit at the declaration makes it easier to identify when quickly reading. There is a similar thing with `extensions` where the former syntax is actually deprecated and slated to be removed with Swift 6.  This just makes everything consistent.